scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -119,6 +120,10 @@ export function SendItem($q,
             scope.metadata = metadata.values;
 
             scope.publish = function() {
+                confirmQuickPublish([scope.item]).then(publishButtonAction);
+            };
+
+            function publishButtonAction() {
```

Comment:
I would inline the code instead of creating a separate function. `scope.publish` is already linked to a button click, so the naming of `publishButtonAction` is not very accurate.

Could you also rename `confirmQuickPublish` and to `confirmPublish`? Because it's no longer only for quick publish.